### DISCUSSION TITLE
How the website was built(https://www.learnpytorch.io)?

### DISCUSSION THREAD

**Discussion by GuangShuaiWang** [mrdbourke/pytorch-deep-learning#395] [Q&A]
I find the website page is very beautiful and readable. And also the website(like chapter 00) is same to Jupyter notebook. so I want ask that if the website was built rely on jupyther notebook? and how to do?

**Comment by mrdbourke** [comment]
Hi @GuangShuaiWang,

Glad you enjoy the website.

It's built with material for MKDocs - https://squidfunk.github.io/mkdocs-material/

As well as `mkdocs-jupyter` - https://github.com/danielfrg/mkdocs-jupyter 

I put these together in a GitHub Action to deploy the site automatically each time a commit is made: https://github.com/mrdbourke/pytorch-deep-learning/blob/main/.github/workflows/make_docs.yml

**Reply by GuangShuaiWang** [reply]
@mrdbourke Thanks for your answer. Besides, I think the contents you write is wonderful.
